app/javascript/application.js

```
@@ -1,3 +1,4 @@
 import './controllers';
-import 'bootstrap';
+import * as bootstrap from 'bootstrap';
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)